src/openms/source/ANALYSIS/ID/SiriusAdapterAlgorithm.cpp

```diff
@@ -312,11 +312,13 @@ namespace OpenMS
       String lib_original = libpath + "/../lib";
       String lib_merged = libpath + "/lib";
       String lib_thirdparty = libpath + "/../../../All/Sirius/lib";
-      String lib_glpk_thirdparty= libpath + "/../../../MacOS/64bit/Sirius/lib" + sep + libpath + "/../../../Linux/64bit/Sirius/lib" + sep +  libpath + "/../../../Windows/64bit/Sirius";
+      String lib_glpk_thirdparty_mac = libpath + "/../../../MacOS/64bit/Sirius/lib"
+      String lib_glpk_thirdparty_lnx = libpath + "/../../../Linux/64bit/Sirius/lib"
+      String lib_glpk_thirdparty_win = libpath;
```

Comment:
I think that does not make sense since the glpk lib is in the Windows/64bit/Sirius directory. And the jar in the All/Sirius/lib path. 

-> libpath + "/../../../Windows/64bit/Sirius"

Reply:
... but it seems to have worked at least for the Adapter. Not sure why the AssayGeneratorMetabo is not working. 
There seems to be something wrong with the -Djava.library.path in this case...

Reply:
I think it works because we are looking for it relative to the executable and not the jar, right? And the executable Sirius...exe is in the Windows subfolder as well.
Yes I also don't have an idea why AGM would work only on macs. I think we would need to debug it on those platforms.

Reply:
Yes, you are right! 
I will try to debug on Unix later today.